docs/14-batch-shipyard-configuration-jobs.md

```diff
@@ -1055,14 +1056,19 @@ property are:
           containers, the specified runtime must be installed and loaded on
           the host compute node.
         * (optional) `options` is a list of options that will be passed to the
-          `mpirun` command.
-        * (optional) `processes_per_node` is the number of processes per node.
-          If the field is specified, options will be automatically passed to
-          the `mpirun` command in accordance with the specified runtime so
-          that `num_instances` * `processes_per_node` processes spawn. If
+          `mpiexec` or `mpirun` (`executable_path`) command.
+        * (optional) `processes_per_node` is either a number that represents
+          the number of processes per node or a command that, when evaluated,
```

Comment:
when evaluated -> when evaluated on the master compute node (or master compute node container)